shogun-gs-interceptor/src/main/java/de/terrestris/shogun/interceptor/config/InterceptorWebSecurityConfig.java

```diff
@@ -19,10 +20,13 @@ protected void customHttpConfiguration(HttpSecurity http) throws Exception {
                 "/v2/**",
                 "/csrf/**"
             )
-            .permitAll()
+                .permitAll()
             .antMatchers("/interceptorrules/**")
                 .hasRole("interceptor-admin")
             .anyRequest()
-                .authenticated();
+                .authenticated()
+            .and()
+                .csrf()
+                .ignoringAntMatchers("/geoserver.action/**");
```

Comment:
And why do we want this?

Reply:
If we want to disable CSRF protection for specfic use cases (e.g. requests via the swagger ui or `curl`), we may want to use a specific matcher (see e.g. [here](https://github.com/terrestris/shogun/pull/40/files#diff-cd40bf5efa26b6504a955a6ad418d59fR14)).